CONTRIBUTING.md

```diff
@@ -19,6 +19,34 @@ The following is a git shell example for installing the app on your local machin
 cd /web_container/
 git clone https://github.com/CenterForDigitalHumanities/deer.git deer
 ```
+### Tomcat installation Guide (Windows)
+
+Note:The Standard Java Development Kit (JDK) needs to be installed for this to work.
+
+Many of the developers use Tomcat9 as their preferred webserver. To avoid combatability issues it's reccomended that new developers use the same or something compatable.
+
+Download Tomcat9 installation wizard from the above link and set up following the installer instrctions. Many people find it usefull to download into the root of a drive or other easily-pathed location.
+
+In powershell or command prompt navigate to the "bin" folder of the tomcat files and run the startup.bat file. This can be done using the following example of pathing commands.
+
+```
+E:\>cd Tomcat9
+
+E:\Tomcat9>cd bin
+
+E:\Tomcat9\bin>startup.bat
+```
+The user then can navigate to the "webapps" folder of the tomcat files. Here a new folder for the DEER project can be created. Navigating to this file in gitbash or windows command prompt is the proper location to clone the git files using the above git shell example.
+
+Note: The name of the folder that the code is cloned into is case sensitive and will become the address of the webserver's localhost address (naming this folder deer will mean that the address will be localhost:####/deer. Naming this folder ROOT or replacing the existing ROOT folder will allow you to navigate to the page without a folder name at the end of the address)
+
+The default port for tomcat9 is 8080, meaning that unless the user has specified a different port for use on this project the page for this project will be http://localhost:8080/deer/ if the folder holding the project's information is named deer.
+
+When working on the project Tomcat9 MUST be opened and started EACH TIME prior to accesing the page. This can be done through the "monitor Tomcat" application now available on your pc.
+
+For testing of components and functionality a test html page should be added to the deer folder root. This  be called test.html and can be viewed when testing by visitin its page at http://localhost:8080/deer/test.html.
+
+An example of a testing html page has been included in the files of the git repository for use. In this testing file the components being tested would be referenced within the body section of the html code. This will change to be whatever component is being worked on
```

Comment:
No, we will not include a test.html in the codebase.  This would be a good spot just to include the example directly, like
```HTML
<html>
    <head>
        <title>Data Encoding and Exhibition for RERUM</title>
        <meta charset="UTF-8">
        <link rel="stylesheet" href="https://unpkg.com/chota@latest">
        <script src="/js/deer.js" type="module"></script>
    </head>

    <body>
        ...
    </body>
</html>
```